### PR TITLE
oom: drop invalid %m in the log message

### PR DESCRIPTION
**Pull request by yuwata**
Fixes https://github.com/systemd/systemd/issues/23785#issuecomment-1210030100.

### CODE REVIEW

**Comment by yuwata** [comment]
cc @evverx and @anitazha

File: src/oom/oomd-manager.c [hunk 1]
```diff
@@ -83,7 +83,7 @@ static int process_managed_oom_message(Manager *m, uid_t uid, JsonVariant *param
 
                         r = cg_path_get_owner_uid(message.path, &cg_uid);
                         if (r < 0) {
-                                log_debug("Failed to get cgroup %s owner uid: %m", message.path);
+                                log_debug("Failed to get cgroup %s owner uid.", message.path);
```
Comment (poettering) [inline src/oom/oomd-manager.c]: maybe instead use `log_debug_errno(r, …`?

Comment (yuwata) [reply src/oom/oomd-manager.c]: Yeah. I may need more caffeine...

**Review by anitazha — APPROVED** [review]
thanks!

**Pull request merged**